tests/testthat/test-install-torsten.R

```diff
@@ -0,0 +1,157 @@
+## Adapted from test-install.R from the cmdstanr package
+
+context("install_torsten")
+
+torsten_test_tarball_url <- Sys.getenv("TORSTEN_TEST_TARBALL_URL")
+if (!nzchar(torsten_test_tarball_url)) {
+  torsten_test_tarball_url <- NULL
+}
+
+test_that("install_torsten() successfully installs torsten", {
+  if (getRversion() < '3.5.0') {
+    dir <- tempdir()
+  } else {
+    dir <- tempdir(check = TRUE)
+  }
```

Comment:
-   I know this is `check = TRUE` part is coming from cmdstanr's tests, but we can drop it and assume that the per-session R temporary directory hasn't been deleted. Much of our code and tests use `tempfile`, and that calls `tempdir` with the default `check = FALSE`, so if something deleted the per-session R temporary directory, our tests aren't going to pass.

-   Ideally this would use a directory *under* `tempdir()` and clean it up afterward. You can use `local_test_dir()` for that, which is a bbr.bayes test helper that wraps `withr::local_tempdir()`.